mmdet/apis/inference.py

```diff
@@ -109,7 +109,7 @@ def inference_detector(model, img):
     else:
         # Use torchvision ops for CPU mode instead
         for m in model.modules():
-            if isinstance(m, (RoIPool, RoIAlign)):
+            if isinstance(m, RoIPool):
```

Comment:
See [here](https://github.com/open-mmlab/mmcv/blob/master/mmcv/ops/roi_pool.py#L71) RoiPool does not have the attribute `aligned`, so it will cause a bug here. We should skip the if condition in line 113.

Reply:
OK. I will skip if condition in [line 113](https://github.com/open-mmlab/mmdetection/blob/d6b1386711b41c21c009f630ab7b5c011a166af6/mmdet/apis/inference.py#L113) and create another PR to add aligned attribute to RoIPool.